src/routes/github/create-branch/github-create-branch-get.ts

```diff
@@ -1,31 +1,34 @@
 import { NextFunction, Request, Response } from "express";
 import { Errors } from "config/errors";
+import { createUserClient } from "utils/get-github-client-config";
 
 // TODO: need to update this later with actual data later on
 const servers = [{ id: 1, server: "http://github.internal.atlassian.com", appName: "ghe-app" }, { id: 2, server: "http://github.external.atlassian.com", appName: "ghe-app-2" }];
-const repos = [{ id: 1, name: "first-repo", org: "org-1" }, { id: 2, name: "second-repo", org: "org-1"  }, { id: 3, name: "third-repo", org: "org-2" }];
 const branches = [{ id: 1, name: "first-branch" }, { id: 2, name: "second-branch" }, { id: 3, name: "third-branch" }];
 
 
 export const GithubCreateBranchGet = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
 	const {
 		jiraHost,
-		githubToken
+		githubToken,
+		gitHubAppConfig
```

Comment:
If we assume `gitHubAppConfig` is available, that means we have the `uuid` present in the url /github/uuid/xxx, and that means we will have the gitHubInstallationId, that means we don't have to use the user token?

Reply:
That would work for Github enterprise, but not for cloud. `uuid` will be undefined for GH cloud users and we won't be able to recognize the exact user.